src/main/resources/hudson/scm/SubversionTagAction/svn-tagform-update-row.js

```diff
@@ -0,0 +1,14 @@
+// update the visual feedback depending on the checkbox state
+function updateRow(changeEvent) {
+    const checkbox = changeEvent.target;
+    checkbox.parentNode.parentNode.style.color = checkbox.checked ? "inherit" : "grey";
+
+    const index = checkbox.getAttribute("data-index");
+    document.querySelector(`input[name="name${index}"]`).disabled = !checkbox.checked;
+}
+
+window.addEventListener("DOMContentLoaded", () => {
+    document.querySelectorAll(".svn-tagform-tag-checkbox").forEach(checkbox => {
+        document.addEventListener("change", updateRow);
```

Comment:
It seems odd to me that we aren't using `checkbox`. If it's truly unused, maybe we should use `()` instead on the previous line? Or if we should use it, should this line be:

```suggestion
        checkbox.addEventListener("change", updateRow);
```

Reply:
Oh, it's an oversight from my side. I'm unsure how this even worked for the sake of the video :/